File: testapp/src/main/java/com/nerdscorner/mvplib/testapp/events/attribute/fragments/view/Fragment1View.java
package com.nerdscorner.mvplib.testapp.events.attribute.fragments.view;

import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;
import android.widget.TextView;

import com.nerdscorner.mvplib.events.bus.Bus;
import com.nerdscorner.mvplib.events.view.BaseFragmentView;
import com.nerdscorner.mvplib.testapp.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class Fragment1View extends BaseFragmentView {
    @BindView(R.id.text) TextView text;

    public Fragment1View(@NonNull Fragment fragment, @NonNull Bus bus) {
        super(fragment, bus);
        ButterKnife.bind(this, fragment.getView());
    }

    public void setText(String text) {
        this.text.setText(text);
    }
}

File: testapp/src/main/java/com/nerdscorner/mvplib/testapp/events/attribute/fragments/view/Fragment1View.kt
package com.nerdscorner.mvplib.testapp.events.attribute.fragments.view

import android.widget.TextView
import androidx.fragment.app.Fragment
import com.nerdscorner.mvplib.events.bus.Bus
import com.nerdscorner.mvplib.events.view.BaseFragmentView
import com.nerdscorner.mvplib.testapp.R
import com.nerdscorner.mvplib.testapp.events.attribute.activities.view.AttributeMainView.ActionClickedEvent

class Fragment1View(fragment: Fragment, bus: Bus) : BaseFragmentView(fragment, bus) {
    var textView: TextView? = fragment.view?.findViewById(R.id.text)

    init {
        onClick(R.id.some_button, ActionClickedEvent())
    }

    fun setText(value: String) {
        textView?.text = value
    }
}
